package.json

```diff
@@ -37,6 +37,11 @@
     "rollup": "^2.30.0"
   },
   "dependencies": {
-    "express": "^4.16.4"
+    "alert": "^5.0.5",
+    "body-parser": "^1.19.0",
+    "express": "^4.16.4",
+    "express-validator": "^6.6.1",
```

Comment:
Looks like this package is not used in the live build and so should not be listed here

```suggestion
```